etna/transforms/math/apply_lambda.py

```diff
@@ -0,0 +1,134 @@
+import warnings
+from typing import Callable
+from typing import Optional
+
+import pandas as pd
+
+from etna.transforms.base import FutureMixin
+from etna.transforms.base import Transform
+from etna.transforms.utils import match_target_quantiles
+
+
+class LambdaTransform(Transform, FutureMixin):
```

Comment:
Remove `FutureMixin`, it should be used when transform can create regressors out of non-regressors.